lib/whitehall_importer/integrity_checker/body_text_check.rb

```diff
@@ -10,11 +10,19 @@ def initialize(proposed_body_text, publishing_api_body_text)
     def sufficiently_similar?
       proposed_body = remove_attachment_file_size(proposed_body_text)
       publishing_api_body = remove_attachment_file_size(publishing_api_body_text)
-      Sanitize.clean(publishing_api_body).squish == Sanitize.clean(proposed_body).squish
+
+      sanitized_proposed_body = remove_accessible_format_notice(Sanitize.clean(proposed_body).squish)
+      sanitized_body = remove_accessible_format_notice(Sanitize.clean(publishing_api_body).squish)
+
+      sanitized_proposed_body == sanitized_body
```

Comment:
I think this might be an opportunity to move the removal of various items from the body text to its own method.  It's possible that we may need to do some additional processing as we fix various body text issues that arise, and this will allow us to add this extra processing cleanly. E.g:

```suggestion
proposed_body = processed_body(proposed_body_text)
publishing_api_body = processed_body(publishing_api_body_text)

Sanitize.clean(proposed_body).squish == Sanitize.clean(publishing_api_body).squish
```

The new method could look like this:
```
def processed_body(body_text)
  processed_body = remove_attachment_file_size(body_text)
  remove_accessible_format_notice(processed_body)
end
```